src/components/booking/SkeletonSessionCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { Skeleton } from "../ui/skeleton";
+
+const SkeletonSessionCard = () => {
+  return (
+    <Skeleton className="rounded-sm border px-6 flex-col gap-1 py-4 min-h-24 flex font-medium bg-secondary opacity-40">
```

Comment:
why the opacity-40 tag?

Reply:
if you mean to apply opacity to the text skeleton you can do *:opacity-40 (applies directly to children) as opacity-40 also lightens the bg-secondary

Reply:
the opacity was added in so that the color matched the disable state of the card but wo/ it also looks good

Reply:
before i looked into in a little bit and shadcn skeleton just fluctuate the opacity so this is kinda like giving a min opacity value

Reply:
*:opacity-40 looks good